src/visumorph/scale.py

```diff
@@ -40,20 +40,16 @@ def scale(image, scale):
     >>> scaled_img = vm.scale(img, 1.15)
     """
 
-    if not isinstance(image, Image):
-        raise TypeError("The provided image must be a valid VisuMorph Image "
-                        "object.")
+    if not isinstance(image, visumorph.Image):
+        raise TypeError("Input 'image' must be a valid VisuMorph Image.")
+    if not isinstance(scale, (int, float)):
+        raise TypeError("Input 'scale' must be a number.")
 
-    if not type(scale) in (float, int):
-        raise TypeError("The scale provided must be a number.")
+    # Calculate new dimensions
+    new_width = round(image.width * scale)
+    new_height = round(image.height * scale)
```

Comment:
What happened when the `new_width` and `new_height` is zero or negative? Also, consider the case of `round(0.01)`, will it result in zero?

Reply:
Resolving this as this line is removed.